app/controllers/api/v2/smart_class_parameters_controller.rb

```diff
@@ -39,7 +41,8 @@ def show
         param :description, String, :desc => N_("Description of smart class")
         param :default_value, String, :desc => N_("Value to use when there is no match")
         param :hidden_value, :bool, :desc => N_("When enabled the parameter is hidden in the UI")
-        param :use_puppet_default, :bool, :desc => N_("Do not send this parameter via the ENC. Puppet will use the value defined in the Puppet manifest for this parameter")
+        param :use_puppet_default, :bool, :desc => N_("Deprecated, please use omit")
+        param :omit, :bool, :desc => N_("Do not send this parameter via the ENC. Puppet will use the value defined in the Puppet manifest for this parameter")
```

Comment:
Shouldn't this (and in the other api controller) be changed to something like "omit from classification output" in line with the UI description?